### PR TITLE
feat: add monitoring resources

### PR DESCRIPTION
**Pull request by alaincormier-ssc**
address #713 for core-landing-zone
(documentation and experimentation in future PR)

- add monitoring resources and permissions
- move storage.admin role for logging namespace service account to logging.yaml

### CODE REVIEW

**Review by fmichaelobrien — APPROVED** [review]
/lgtm

**Review by cartyc — APPROVED** [review]

**Review by lucstjean-ssc — APPROVED** [review]
/lgtm

File: solutions/core-landing-zone/namespaces/logging.yaml [hunk 1]
```diff
@@ -44,6 +44,60 @@ spec:
...
+spec:
+  resourceRef:
+    apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+    kind: Project
+    external: logging-project-id # kpt-set: ${logging-project-id}
+  # AC-3(7), AC-3, AC-16(2)
+  role: roles/monitoring.admin
+  member: "serviceAccount:logging-sa@management-project-id.iam.gserviceaccount.com" # kpt-set: serviceAccount:logging-sa@${management-project-id}.iam.gserviceaccount.com
+---
+# Grant GCP role Storage Admin to GCP SA
```
Comment (davelanglois-ssc) [inline solutions/core-landing-zone/namespaces/logging.yaml:83]: I would add "on logging project"

Comment (alaincormier-ssc) [reply solutions/core-landing-zone/namespaces/logging.yaml]: comment updated
metadata.name also updated to be consistent with names of other similar objects

**Review by davelanglois-ssc — APPROVED** [review]

**Review by fmichaelobrien — APPROVED** [review]
/lgtm

**Pull request merged**